lib/storage/bucket.js

```diff
@@ -864,9 +864,14 @@ Bucket.prototype.setMetadata = function(metadata, callback) {
 };
 
 /**
- * Upload a file to the bucket. This is a convenience method that wraps the
- * functionality provided by a File object, {module:storage/file}.
+ * Upload a file to the bucket. This is a convenience method that wraps
+ * {module:storage/file#createWriteStream}.
  *
+ * Resumable uploads are automatically enabled if your input file is larger than
+ * 5 MB. You can override this logic by specifying `options.resumable` as `true`
```

Comment:
Got it.

I think I found this confusing because we lead with how resumable uploads are auto-enabled, so the sentence after feels directly related to that?

What if we did something like..

You can specify whether or not an upload is resumable by setting `options.resumable` to `true` or `false`. *Resumable uploads are enabled by default if your input file is larger than 5 MB.*